api/editor/api-editor/src/main/java/ca/bc/gov/tno/areas/admin/controllers/SeriesController.java

```diff
@@ -0,0 +1,99 @@
+package ca.bc.gov.tno.areas.admin.controllers;
+
+import java.util.List;
+
+import javax.annotation.security.RolesAllowed;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import ca.bc.gov.tno.dal.db.services.interfaces.ISeriesService;
+import ca.bc.gov.tno.dal.db.entities.Series;
+
+/**
+ * Endpoints to communicate with the TNO DB series.
+ */
+@RolesAllowed({ "administrator" })
+@RestController("AdminSeriesController")
+@RequestMapping({ "/admin/series", "/api/admin/series" })
+public class SeriesController {
```

Comment:
Now we can get series.

Reply:
Do we know if series is going to be something manually added or if we have a base set of them that we will have scripts for? Just something I thought about when I added random ones for testing via SQL

Reply:
They have a list will need to import.  But they also need to have an admin for them.